communication/state.cpp

```diff
@@ -74,17 +74,6 @@ State::State(Battery& battery, state_conf_t config):
 
     mav_mode_custom = config.mav_mode_custom;
 
-    simulation_mode = config.simulation_mode;
-
-    if (simulation_mode == HIL_ON)
-    {
-        mav_mode |= MAV_MODE_FLAG_HIL_ENABLED;
```

Comment:
@jlecoeur 
We still need to know from QGC that we are in HIL mode,
where shall it be done then ?